src/main/java/ca/corefacility/bioinformatics/irida/ria/web/models/export/NcbiSubmissionRequest.java

```diff
@@ -0,0 +1,64 @@
+package ca.corefacility.bioinformatics.irida.ria.web.models.export;
+
+import java.util.Date;
+import java.util.List;
+
+/**
+ * Request for submitting to the NCBI SRA
+ */
+public class NcbiSubmissionRequest {
+	private Long projectId;
+	private String bioProject;
```

Comment:
`bioProject` is just the name of the bio project correct? If so let's rename this to bioProjectName or something similar for clarity. I've never used NCBI but if it expects the variable to be named bioProject then we can leave it alone

Reply:
I am just going off the naming convention that was already established.

Reply:
Sounds good. Thanks!